.github/workflows/framework-kind.yml

```
@@ -0,0 +1,14 @@
+name: Framework
+
+on:
+  push:
+    tags:
+      - v0.**
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  fw-kind-tests:
+    name: KinD
+    uses: stolostron/governance-policy-framework/.github/workflows/kind.yml@main
```

nit: trailing newline

```suggestion
    uses: stolostron/governance-policy-framework/.github/workflows/kind.yml@main

```